localization/languages/fa.json

```diff
@@ -5,25 +5,25 @@
     /* Context menu items
         these are the items displayed in the menu when you right-click on a page
         */
-    "addToDictionary": null, //missing translation
+    "addToDictionary": "افزودن به دیکشنری",
     "pictureInPicture": null, //missing translation
     "openInNewTab": "باز کردن در برگه جدید",
     "openInNewPrivateTab": "باز کردن در برگه خصوصی جدید",
-    "saveLinkAs": null, //missing translation
+    "saveLinkAs": "ذخیره لینک به عنوان",
     "viewImage": "نمایش تصویر",
-    "openImageInNewTab": "باز کردن تصویر در برگه جدید",
-    "openImageInNewPrivateTab": "باز کردن تصویر در برگه خصوصی جدید",
-    "saveImageAs": "ذخیره تصویر",
+    "openImageInNewTab": "نمایش تصویر در برگه جدید",
+    "openImageInNewPrivateTab": "نمایش تصویر در برگه خصوصی جدید",
+    "saveImageAs": "ذخیره تصویر به عنوان",
     "searchWith": "جستجو با %s", //%s will be replaced with the name of the current search engine
-    "copyLink": "کپی لینک",
-    "copyEmailAddress": null, //missing translation
-    "selectAll": null, // Missing Translation
-    "undo": null, // Missing Translation
+    "copyLink": "کپی پیوند",
+    "copyEmailAddress": "کپی رایانامه",
+    "selectAll": "انتخاب همه", // this is farsi, but in persian we say گزینش همه
```

Comment:
What translation does Chrome use here? We can probably use the same one.

Reply:
ok, i searching for them in my friends :+1: